### PR TITLE
JDK-8195809: [TESTBUG] jps and jcmd -l support for containers is not tested

### PR DESCRIPTION
**Pull request by mseledts**
Please review this addition of new testing:
  - new test for sending jcmd commands across container boundaries
  - updated existing sidecar test, enabling jcmd test case

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8195809](https://bugs.openjdk.java.net/browse/JDK-8195809): [TESTBUG] jps and jcmd -l support for containers is not tested

### Reviewers
 * [Harold Seigel](https://openjdk.java.net/census#hseigel) (@hseigel - **Reviewer**) ⚠️ Review applies to 1a66f1c0ea99667c2df9074a6a042069a131db9d
 * [Kevin Walls](https://openjdk.java.net/census#kevinw) (@kevinjwalls - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/5368/head:pull/5368` \
`$ git checkout pull/5368`

Update a local copy of the PR: \
`$ git checkout pull/5368` \
`$ git pull https://git.openjdk.java.net/jdk pull/5368/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 5368`

View PR using the GUI difftool: \
`$ git pr show -t 5368`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/5368.diff">https://git.openjdk.java.net/jdk/pull/5368.diff</a>

</details>

### CODE REVIEW

**Comment by mseledts** [comment]
@hseigel and Kevin Walls, please review when you have a chance

File: test/hotspot/jtreg/containers/docker/TestJcmd.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            testVmInfo(pid);
+
+            p.waitFor();
+        } finally {
+            DockerTestUtils.removeDockerImage(IMAGE_NAME);
+        }
+    }
+
+
+    // Run "jcmd -l" in a sidecar container, find the target process.
```
Comment (hseigel) [inline test/hotspot/jtreg/containers/docker/TestJcmd.java:94]: Since 'jcmd -l' is not run in a sidecar, this comment is incorrect.

Comment (mseledts) [reply test/hotspot/jtreg/containers/docker/TestJcmd.java:94]: Good catch. I will update the comment.

**Review by hseigel — APPROVED** [review]
The changes look good!  Thanks for doing this.
Harold

**Review by kevinjwalls — APPROVED** [review]
Yes looks good. I've run the tests and both running OK and passing.

**Comment by mseledts** [comment]
Harold, Kevin, thank you for review.

/integrate

**Pull request closed without merging**